starter-code/lib/harryPotter.js

```diff
@@ -1,3 +1,30 @@
+var _ = require('lodash');
```

Comment:
Vamos chicos!!! Seguid trabajando. Si tenéis dudas.... no nos preguntéis xD xD Ahora en serio, dadle duro.